src/tests/JIT/Performance/JittedMethodsCountingTest/JittedMethodsCountingTest.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="JittedMethodsCountingTest.cs" />
+  </ItemGroup>
```

Comment:
Are there configurations that this test needs to be disabled for? I am thinking about JIT stress, etc.

Reply:
Yeah, adding `<JitOptimizationSensitive>true</JitOptimizationSensitive>` to the `PropertyGroup` seems like a good idea.